Syntaxes/Asciidoctor.sublime-syntax

```diff
@@ -7,20 +7,6 @@
 # Syntax definition for Asciidoctor:
 # -- https://asciidoctor.org/
 # ------------------------------------------------------------------------------
-# NOTE: This syntax was taken from @bsmith-n4, who slightly improved the
```

Comment:
If we remove this note we should then credit @bsmith-n4 in the main README. I'll create an Issue to remind me of this task, so that wen we merge all the changes into `master` the due credits are there also.

Reply:
Done, see Issue #25.